rust_src/remacs-sys/lib.rs

```diff
@@ -1554,13 +1684,15 @@ extern "C" {
     pub fn unchain_marker(marker: *mut Lisp_Marker);
     pub fn del_range(from: ptrdiff_t, to: ptrdiff_t);
     pub fn buf_bytepos_to_charpos(b: *mut Lisp_Buffer, bytepos: ptrdiff_t) -> ptrdiff_t;
+    pub fn swap_in_symval_forwarding(sym: *mut Lisp_Symbol, blv: *mut Lisp_Buffer_Local_Value);
+
 }
 
 /// Contains C definitions from the font.h header.
 pub mod font {
     use libc::c_int;
 
-    /// Represents the indices of font properties in the contents of a font
+    /// Represents // TODO: he indices of font properties in the contents of a font
```

Comment:
Typo?

Reply:
Thanks for catching that. A yasnippet miss fire when fixing indentation.